src/components/Card/Card.css

```
@@ -1,5 +1,24 @@
 .Card {
-  color: var(--color-primary);
-  border-bottom: 1px solid;
-  padding: 12px;
+  padding: 17px 20px;
+  border-radius: 4px;
+  box-shadow: 7px 7px 5px -6px rgba(17, 17, 17, 0.5);
+  margin: 20px;
+}
+
+.names {
```

I'd recommend using BEM. Also, this is one name and not multiple.
```suggestion
.Card__name {
```